app/models/list_set.rb

```diff
@@ -40,31 +40,29 @@ def lists
   def a_to_z_list
     [ListSet::List.new(
       "A to Z",
-      content_tagged_to_tag.reject do |content|
-        excluded_formats.include? content.format
-      end.sort_by(&:title)
+      content_tagged_to_tag
+        .reject { |content| excluded_formats.include? content.format }
+        .sort_by(&:title)
     )]
   end
 
   def curated_list
-    @group_data.map do |group|
-      contents = group["contents"].map do |base_path|
-        content_tagged_to_tag.find do |content|
+    @group_data.map { |group|
+      contents = group["contents"].map { |base_path|
+        content_tagged_to_tag.find { |content|
           content.base_path == base_path
-        end
-      end.compact
+        }
+      }.compact
 
       ListSet::List.new(group["name"], contents) if contents.any?
-    end.compact
+    }.compact
```

Comment:
Thanks - I've introduced a variable and moved one of the `compact` calls inline.

(As an aside, I personally quite like the convention that you should use braces for expressions which are functional (i.e. map or filter something with no side effects), and `do..end` for expressions which, say, modify state or perform validation, because it makes the latter stand out as potentially dangerous. But I'll stick with the single/multiline convention if that's what you normally use. 🙂)